### PR TITLE
Revert "[6.0][Tests][REPL] Disable some REPL tests on aarch64 Linux."

### PR DESCRIPTION
**Pull request by al45tair**
Reverts apple/llvm-project#8749

### CODE REVIEW

**Comment by al45tair** [comment]
@swift-ci Please test

**Review by adrian-prantl — APPROVED** [review]

**Comment by al45tair** [comment]
@swift-ci Please test Windows platform

**Comment by al45tair** [comment]
@adrian-prantl I can't merge this as I don't have commit rights.

**Pull request merged**
